packages/cli/src/commands/deploy/index.ts

```diff
@@ -20,7 +21,31 @@ export default async client => {
   let argv = null;
 
   try {
-    argv = getArgs(client.argv.slice(2), args);
+    argv = getArgs(client.argv.slice(2), {
```

Comment:
Instead of inlining this code, you can rename `./packages/cli/src/commands/deploy/args.js` to `.ts` and append `as const` after the args.

Reply:
For some reason that not understand, When I opt for your solution this happens @styfle 

<img width="1272" alt="Screen Shot 2021-06-28 at 9 53 29" src="https://user-images.githubusercontent.com/16585386/123657771-b7c50080-d7f6-11eb-8421-52c3c8258486.png">

Log error

`
Argument of type '{ '--force': BooleanConstructor; '--with-cache': BooleanConstructor; '--public': BooleanConstructor; '--no-clipboard': BooleanConstructor; ... 15 more ...; '--target': StringConstructor; }' is not assignable to parameter of type 'Spec'.
  Property ''--env'' is incompatible with index signature.
    Type 'StringConstructor[]' is not assignable to type 'string | Handler | [Handler]'.
      Type 'StringConstructor[]' is not assignable to type '[Handler]'.
        Target requires 1 element(s) but source may have fewer.ts(2345)
`